zeroex/orderwatch/watcher.go

```diff
@@ -492,7 +494,7 @@ func (w *Watcher) rewatchOrder(order *meshdb.Order, orderInfo *zeroex.OrderInfo)
 	w.expirationWatcher.Add(order.SignedOrder.ExpirationTimeSeconds.Int64(), order.Hash)
 }
 
-func (w *Watcher) unwatchOrder(order *meshdb.Order) {
+func (w *Watcher) unwatchOrder(order *meshdb.Order, didExpire bool) {
```

Comment:
We should be careful not to fall into the [boolean flag trap](https://ariya.io/2011/08/hall-of-api-shame-boolean-trap). I'm okay with it for now because it is fairly isolated in scope.